src/algebra/lie/of_associative.lean

```diff
@@ -66,6 +66,31 @@ lemma of_associative_ring_bracket (x y : A) : ⁅x, y⁆ = x*y - y*x := rfl
 
 end lie_ring
 
+section associative_module
+
+variables {M : Type w} [add_comm_group M] [module A M]
+
+/-- We can regard a module over an associative ring `A` as a Lie ring module over `A` with Lie
+bracket equal to its ring commutator.
+
+Note that this cannot be a global instance because it would create a diamond when `M = A`,
+specifically we can build two mathematically-different `has_bracket A A`s:
+ 1. `@ring.has_bracket A _` which says `⁅a, b⁆ = a * b - b * a`
+ 2. `(@lie_ring_module.of_associative_module A _ A _ _).to_has_bracket` which says `⁅a, b⁆ = a • b`
+    (and thus `⁅a, b⁆ = a * b`) -/
+def lie_ring_module.of_associative_module : lie_ring_module A M :=
```

Comment:
```suggestion
    (and thus `⁅a, b⁆ = a * b`)

See note [reducible non-instances] -/
@[reducible
def lie_ring_module.of_associative_module : lie_ring_module A M :=
```
def lie_ring_module.of_associative_module : lie_ring_module A M :=
```

Reply:
Thanks, I had not read that library note before.